sourcetools/j9nls/com/ibm/oti/NLSTool/J9NLS.java

```diff
@@ -138,6 +153,13 @@ private void generatePropertiesFiles(String outFileName, boolean palmMode) throw
 		Vector defaultNLSFiles = (Vector) localeHashtable.get(DEFAULT_LOCALE);
 		StringBuffer buffer = new StringBuffer();
 
+		// Create the output directory if it doesnt exist
```

Comment:
Does the nls directory always exist with the old usage of the tool?

Reply:
Yes since the input dir == output dir. If it doesnt exist, you will fail earlier when you are scanning for input files in `searchNLSFiles()`